### DISCUSSION TITLE
内存使用过高，不释放问题

### DISCUSSION THREAD

**Discussion by Mrkaixuanwang** [taosdata/TDengine#10468] [Q&A]
环境配置：
1.服务器配置：arm架构 6核  8G内存
1.使用TDengine:2.2.0.5镜像
2.数据库配置：blocks 3； cache 1M；vgroups 1
在TDengine使用过程中发现如下问题：
在数据库启动后，数据库大概占用0.3%即24M内存；随后启动应用程序，此时数据库占用的内存会慢慢增高至15%即1.2G的内存；当把应用服务关掉后，通过一段时间的观察，数据库占用的内存并不会下降。
想了解这些内存是如何占用的？根据官网文档，在数据写入时，数据会缓存在内存中，但根据配置，缓存只应占用3*1M的内存。但通过对blocks和cache的配置并不能减少内存的占用。应用服务大概创建了350张表。另外是否可以通过配置减少内存的使用？

**Comment by fenghuazzm** [comment]
可以加这个微信：15652223354
我们排查一下
